x/dclauth/keeper/msg_server_approve_revoke_account.go

```diff
@@ -55,6 +55,18 @@ func (k msgServer) ApproveRevokeAccount(goCtx context.Context, msg *types.MsgApp
 
 	// check if pending account revocation has enough approvals
 	if len(revoc.Approvals) == k.AccountApprovalsCount(ctx) {
+		// get account
+		account, ok := k.GetAccountO(ctx, sdk.AccAddress(accAddr))
+
+		// check we can get that account or can not
+		if !ok {
+			return nil, types.ErrAccountDoesNotExist(msg.Address)
+		}
+
+		// save to map RevokedAccount
```

Comment:
Better comment would be `create revoked account record`

Reply:
Done! ✅